editoast/src/views/search/sqlquery.rs

```diff
@@ -80,26 +82,22 @@ impl SqlQuery {
     }
 
     /// Builds the SQL code represented by self
-    pub fn to_sql(&self, bind_pos: &mut u64) -> String {
+    pub fn to_sql(&self, bind_pos: &mut u64, string_bind_map: &mut HashMap<u64, String>) -> String {
```

Comment:
We can use a `Vec` instead of an `HashMap`. It is algorithmically better and we don't need to pass a `&mut u64` since the length of the `Vec` can be used. The only drawback is the gap in the indexes (the first index will be `0` instead of `1`).